Sources/PKYouboraAdsAdapter.swift

```diff
@@ -198,6 +234,7 @@ extension PKYouboraAdsAdapter {
                 messageBus.addObserver(self, events: [e.self]) { [weak self] event in
                     guard let self = self else { return }
                     self.fireStop(["skipped":"true"])
+                    self.tryAdBreakStop()
```

Comment:
skip should result in adBreakEnded, why do we need to have try logic all around?

Reply:
The explanation above